### ISSUE TITLE
[BUG] Link broken in "Authentication in Azure Management Libraries for Java" README

### ISSUE THREAD

**Issue by vcolin7** [Azure/azure-libraries-for-java#1169]
**Describe the bug**
There is a dead link in the **Authentication in Azure Management Libraries for Java** [README](https://github.com/Azure/azure-libraries-for-java/blob/master/AUTH.md#using-an-authentication-file) in the **Using an Authentication File** section where it reads "this sample".

**Expected behavior**
The link should lead to a valid code sample.

**Information Checklist**
Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [X] Bug Description Added
- [ ] Repro Steps Added
- [ ] Setup information Added

**Comment by yungezz** [comment]
HI @xccc-msft could you pls fix the link? thanks

**Comment by xccc-msft** [comment]
@vcolin7 I didn't get your query. The code sample is provided as
`Azure azure = Azure.authenticate(credFile).withDefaultSubscription();`
and
`Azure azure = Azure.authenticate(new File("my.azureauth")).withDefaultSubscription();`

By the description `To learn more about authentication in the Azure Libraries for Java, see AUTH.md.`, it should be clear that it's a link with more details regarding how to prepare/generate the authentication file.

**Comment by vcolin7** [comment]
No problem, let me rephrase: In [this README](**url**), the last paragraph in the [Using an authentication file](https://github.com/Azure/azure-libraries-for-java/blob/master/AUTH.md#using-an-authentication-file) section contains a two links. The [first one](https://github.com/Azure/azure-sdk-for-java/blob/master/azure-samples/src/main/java/com/microsoft/azure/management/graphrbac/samples/ManageServicePrincipal.java) (where it reads `create an auth file by following this sample.`) doesn't take you anywhere.

**Comment by xccc-msft** [comment]
@vcolin7 Thanks for clarification. I just created PR to update this link to [sample of auth file creation](https://github.com/Azure-Samples/aad-java-manage-service-principals/blob/master/src/main/java/com/microsoft/azure/management/graphrbac/samples/ManageServicePrincipal.java).
